modules/mediagoBidAdapter.js

```diff
@@ -46,6 +47,40 @@ function getRandomId(
     );
 }
 
+
+/**
+ * @param {string} key
+ * @param {string} value
+ * @param {string} [expires='']
+ * @param {string} [sameSite='/']
+ * @param {string} [domain] domain (e.g., 'example.com' or 'subdomain.example.com').
+ * If not specified, defaults to the host portion of the current document location.
+ * If a domain is specified, subdomains are always included.
+ * Domain must match the domain of the JavaScript origin. Setting cookies to foreign domains will be silently ignored.
+ */
+const setCookie = function(key, value, expires, sameSite, domain) {
+  if (getProperty(window, 'document', 'cookie')) {
+    const domainPortion = (domain && domain !== '') ? ` ;domain=${encodeURIComponent(domain)}` : '';
+    const expiresPortion = (expires && expires !== '') ? ` ;expires=${expires}` : '';
+    const isNone = (sameSite != null && sameSite.toLowerCase() == 'none')
+    const secure = (isNone) ? '; Secure' : '';
+    document.cookie = `${key}=${encodeURIComponent(value)}${expiresPortion}; path=/${domainPortion}${sameSite ? `; SameSite=${sameSite}` : ''}${secure}`;
+  }
+};
+
+/**
+ * @param {string} name
+ * @returns {(string|null)}
+ */
+const getCookie = function(name) {
+  if (getProperty(window, 'document', 'cookie')) {
+    let m = window.document.cookie.match('(^|;)\\s*' + name + '\\s*=\\s*([^;]*)\\s*(;|$)');
```

Comment:
why did you comment out and bypass storage manager?

Reply:
I need to compatible with Prebid v2.0 and v3.0. 
And the storage manager dosen't be added in v2.0 and v3.0.